pages/linux/mkswap.md

```diff
@@ -1,20 +1,17 @@
 # mkswap
 
-> Sets up a Linux swap area on a device or in a file.
+> Set up a Linux swap area on a device or in a file.
+> Note: `path/to/file` can either point to a regular file or a swap partition.
```

Comment:
```suggestion
> Note: `{{path/to/file}}` can either point to a regular file or a swap partition.
```

Reply:
Why put placeholders in the description? Does any client highlight them?

Reply:
Just now checked it, seems like it's not the case, I have seen pages highlighting placeholders too so anything is fine IG, will mark this as resolved.